krux_boto/util.py

```diff
@@ -43,6 +43,25 @@ def get_instance_region():
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
 
+def setup_hosts(hosts, accepted_domains=['krxd.net'], default='krxd.net'):
```

Comment:
Oh yeah, good catch, @jib. Let's turn these into constants.

@Andrew-peng, you want to use two different constants here. So that someone can change one without the other. Also, it makes better sense conceptually.